### ISSUE TITLE
Steps to phase 4 ("Standardize the Feature")

### ISSUE THREAD

**Issue by binji** [WebAssembly/sign-extension-ops#2]
See https://github.com/WebAssembly/meetings/blob/master/process/phases.md#4-standardize-the-feature-working-group.

We discussed this in the March 21st CG meeting. There are now two implementations (in Chrome and Firefox) behind flags, as well as toolchain support in Emscripten. The spec and reference interpreter have also been updated.

There was some concern about ensuring that the tests are being run by both implementations, and are consistent. @dschuff confirmed that he had tested the implementation in Chrome when this proposal was part of the Threads proposal, and assumed that @kripken had performed similar tests in Firefox.

It might be worth having stronger confirmation that the core tests are being tested by both implementations however. We should try to do this before the next CG meeting.

**Comment by lars-t-hansen** [comment]
We've had our own test suite for these operations, but I've imported the proposal's tests today and run them, and we pass all of them.  They'll land this week and then we'll have coverage on all platforms.

We're tracking work items to land this feature in https://bugzil.la/1429818.

I have a modest proposal that is perhaps more relevant to other in-development features than to this one.  It is that we not mix test cases for in-development features in with the existing test cases, but instead develop tests in new .wast files.  We can, if desirable, merge those files into existing files once the feature is ready to land.  The reason for this is so that when we import test cases from the spec & proposal repos into our individual implementations, we do not risk getting other changes to test cases than the new test cases for the feature we're looking at.

**Comment by lars-t-hansen** [comment]
By the way... is anyone planning to ship this for secure contexts only?

**Comment by lars-t-hansen** [comment]
I should add, our tentative inclination is to ship it for all contexts (small feature that is basically an adjustment to the ISA; grandfathered in).

**Comment by binji** [comment]
> I have a modest proposal that is perhaps more relevant to other in-development features than to this one. It is that we not mix test cases for in-development features in with the existing test cases, but instead develop tests in new .wast files.

Yes, I think this makes sense, as long as they are combined later for clarity. @rossberg, any thoughts?
